src/openslide-vendor-dicom.c

```diff
@@ -698,6 +700,100 @@ static bool maybe_add_file(openslide_t *osr,
   }
 }
 
+struct property_iterate {
+  openslide_t *osr;
+  const char *prefix;
+};
+
+static bool add_properties_element(const DcmElement *element,
+                                   void *client);
+
+static bool add_properties_dataset(const DcmDataSet *dataset,
+                                   void *client) {
+  return dcm_dataset_foreach(dataset, add_properties_element, client);
+}
+
+static bool add_properties_element(const DcmElement *element,
+                                   void *client) {
+  struct property_iterate *iter = (struct property_iterate *) client;
+  DcmVR vr = dcm_element_get_vr(element);
+  uint32_t tag = dcm_element_get_tag(element);
+  const char *keyword = dcm_dict_keyword_from_tag(tag);
+  DcmVRClass klass = dcm_dict_vr_class(vr);
```

Comment:
I don't feel strongly about it, but note that `class` isn't a reserved word in C.

Reply:
I've had many problems with clang interpreting C++ reserved words in plain C mode :( One of the most annoying was `or`, there's an open clang bug on this right now.

It's safest to avoid them I think.

Reply:
Ah, it was clang-format rather than clang, but the issue is the same:

https://github.com/llvm/llvm-project/issues/59188